roles/networking_mapper/molecule/default/vars/internalapi_output.yml

```diff
@@ -1,6 +1,6 @@
 ---
-internal-api:
-  dnsDomain: internal-api.example.com
+internalapi:
+  dnsDomain: internalapi.example.com
```

Comment:
We don't really need this line, as the mapper generated the search domain based on the network name. Just FYI, no need to remove it here, as it's a molecule.

Reply:
that's the output, yes. so it's in, and has to be nudged in the generated snippet used for the product deploy :).